MdeModulePkg/Bus/Pci/PciBusDxe/PciDeviceSupport.c

```diff
@@ -15,6 +15,83 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 //
 LIST_ENTRY  mPciDevicePool;
 
+// MU_CHANGE Begin - Disable BME at Exit Boot Services
+
+/**
+ * DisableBmeOnTree
+ *
+ * @param Bridge
+ *
+ * @return VOID
+ */
+VOID
+DisableBmeOnTree (
+  IN LIST_ENTRY  *Head
+  )
+{
+  LIST_ENTRY     *CurrentLink;
+  PCI_IO_DEVICE  *PciIoDevice;
+  UINT16         Command;
+
+  // DisableBME on ExitBootServices should be synchonized with VTd disable - with
+  // DisableBME running before VTd disable.  If the VTd disable code runs at TPL_CALLBACK,
+  // then DisableBME will run before VTd disable.
+  if (!PcdGetBool (PcdDisableBMEonEBS)) {
+    DEBUG ((EFI_D_WARN, "%a PCI Bus Master enabled due to PCD setting\n", __FUNCTION__));

```

Comment:
Recommendation (feel free to ignore) is to update to `__func__` while adding changes.

```suggestion
DEBUG ((EFI_D_WARN, "%a PCI Bus Master enabled due to PCD setting\n", __func__));
```

Reply:
Fixed.